### PR TITLE
[DO-2386][Contains migration] Provide full localization for KVED register

### PR DESCRIPTION
**Pull request by Andrii17s**
https://razom.atlassian.net/browse/DO-2386
According to the attachment document save translation for each column in the register

### CODE REVIEW

**Review by EmmaSamoilova — APPROVED** [review]

**Review by AlenaYanish — APPROVED** [review]

File: business_register/management/commands/fill_eng_kveds.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    def handle(self, *args, **options):
+        if not args:
+            args = ['KVED_SECTIONS', 'KVED_GROUPS', 'KVED_DIVISIONS', 'KVEDS']
+        wb = load_workbook('kveds.xlsx')
+        for arg in args:
+            sheet_obj = wb[arg]
+            data = {}
+            for row in range(1, sheet_obj.max_row):
+                data[sheet_obj.cell(row=row, column=1).value] = sheet_obj.cell(row=row, column=2).value
+            print(arg, KvedSection.objects.all())
```
Comment (roman-tiukh) [inline business_register/management/commands/fill_eng_kveds.py:19]: print?

File: business_register/management/commands/fill_eng_kveds.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+#from business_register.constants import KVED_SECTIONS, KVEDS, KVED_DIVISIONS, KVED_GROUPS
+from business_register.models.kved_models import KvedDivision, KvedSection, KvedGroup, Kved
+
+
+class Command(BaseCommand):
+    help = 'Fill kved name_en field'
+
+    def handle(self, *args, **options):
+        if not args:
+            args = ['KVED_SECTIONS', 'KVED_GROUPS', 'KVED_DIVISIONS', 'KVEDS']
```
Comment (roman-tiukh) [inline business_register/management/commands/fill_eng_kveds.py:12]: How you can set these args?
Why not make command arguments?

File: business_register/management/commands/fill_eng_kveds.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        wb = load_workbook('kveds.xlsx')
+        for arg in args:
+            sheet_obj = wb[arg]
+            data = {}
+            for row in range(1, sheet_obj.max_row):
```
Comment (roman-tiukh) [inline business_register/management/commands/fill_eng_kveds.py:17]: you can use `iter_rows` for this

File: business_register/management/commands/fill_eng_kveds.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+                data[sheet_obj.cell(row=row, column=1).value] = sheet_obj.cell(row=row, column=2).value
+            print(arg, KvedSection.objects.all())
+            if arg == 'KVED_SECTIONS':
+                for section in KvedSection.objects.all():
+                    section.name_en = data.get(section.name, '')
+                    section.save(update_fields=['name_en'])
+            if arg == 'KVED_GROUPS':
```
Comment (roman-tiukh) [inline business_register/management/commands/fill_eng_kveds.py:24]: elif

File: business_register/management/commands/fill_eng_kveds.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+                for group in KvedGroup.objects.all():
+                    group.name_en = data.get(group.name, '')
+                    group.save(update_fields=['name_en'])
+            if arg == 'KVED_DIVISIONS':
```
Comment (roman-tiukh) [inline business_register/management/commands/fill_eng_kveds.py:28]: elif

File: business_register/management/commands/fill_eng_kveds.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+                for division in KvedDivision.objects.all():
+                    division.name_en = data.get(division.name, '')
+                    division.save(update_fields=['name_en'])
+            if arg == 'KVEDS':
```
Comment (roman-tiukh) [inline business_register/management/commands/fill_eng_kveds.py:32]: elif

File: business_register/management/commands/fill_eng_kveds.py [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+from business_register.models.kved_models import KvedDivision, KvedSection, KvedGroup, Kved
+
+
+class Command(BaseCommand):
+    help = 'Fill kved name_en field'
+
+    def handle(self, *args, **options):
+        if not args:
+            args = ['KVED_SECTIONS', 'KVED_GROUPS', 'KVED_DIVISIONS', 'KVEDS']
+        wb = load_workbook('kveds.xlsx')
```
Comment (roman-tiukh) [inline business_register/management/commands/fill_eng_kveds.py:13]: Don't put your files at the root of the project...
Look at other similar cases.

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
Update migration pls. For now, we have 0162 as the maximum migration number

**Pull request closed without merging**